justfile

```diff
@@ -262,6 +262,39 @@ wait-for-electrs-to-be-ready:
         fi
     done
 
+[private]
+wait-for-coordinator-to-be-ready:
+    #!/usr/bin/env bash
+    set +e
+
+    endpoint="http://localhost:8000/api/newaddress"
+    max_attempts=10
+    sleep_duration=1
+
+    check_endpoint() {
+      response=$(curl -s -o /dev/null -w "%{http_code}" "$endpoint")
+      if [ "$response" -eq 200 ]; then
+        echo "Endpoint is ready!"
+        exit 0
+        else
+        echo "Endpoint not ready yet. Retrying..."
```

Comment:
🙃 It's helpful if the two `echo`s in this recipe mention the coordinator and not just an "endpoint".